blockdag/reachability.go

```diff
@@ -332,10 +343,12 @@ func (rtn *reachabilityTreeNode) countSubtrees() uint64 {
 
 // propagateInterval applies new intervals using a BFS traversal.
 // The intervals are allocated according to subtree sizes and the
-// 'split' allocation rule (see the split() method for further
+// 'split' allocation rule (see the splitExponential() method for further
```

Comment:
I guess you wanna change 'split' to 'splitExponential' or the new name of the function